### ISSUE TITLE
Get rid of "lwaftr generator" and use "packetblaster lwaftr" instead

### ISSUE THREAD

**Issue by dpino** [Igalia/snabbswitch#503]
`lwaftr generator` is a tool for generating traffic for lwaftr softwires. After the latest changes the tool is broken. I got a fix for that but it needs more work (after the fix IPv6 packets are not generated OK). Point is, everything this tool does can be done with `packetblaster lwaftr`.

`packetblaster lwaftr` generates traffic for lwaftr softwires too. It can print out its output to a pcap file, like `lwaftr generator` does. `lwaftr generator` can generate _from-inet_ and _from-b4_ traffic, `packetblaster lwaftr` generates traffic for both directions. If we want to split out traffic we just need to postprocess the generated .pcap file with `tcpdump`. Similarly, if we want to generate VLAN packets we can postprocess the .pcap file with `tcpreplay`.

There's one thing that perhaps needs to be modified in packeblaster lwaftr: by default it generates IMIX traffic of several sizes. We usually want 550-byte packets, all the same size, so adding a --packet-size knob with sizes separated by comma could solve that.

Most of the business logic of `packetblaster lwaftr` is in `apps/test/lwaftr.lua`. I think this file is only referenced by packetblaster. IMHO it should be moved to `packetblaster/lwaftr` or `apps/lwaftr`.

Summarizing:

**lwaftr.conf**

```
$ cat snabbvmx-lwaftr-xe0.conf
vlan_tagging = false,
binding_table = binding_table_60k.txt.s,
aftr_ipv6_ip = 2001:db8::1,
aftr_ipv4_ip = 172.20.1.16,
aftr_mac_inet_side = 02:42:df:27:05:00,
inet_mac = 02:02:02:02:02:02,
aftr_mac_b4_side = 02:42:df:27:05:00,
next_hop6_mac = 02:02:02:02:02:02,
ipv4_mtu = 9000,
ipv6_mtu = 9000
```

``` bash
$ sudo ./snabb lwaftr generator --max-packets 60000 \
   --from-inet --pcap from-inet.pcap lwaftr.conf 193.5.1.100 6
```

Same as:

``` bash
$ sudo ./snabb packetblaster lwaftr \
   --src_mac 02:02:02:02:02:02 --dst_mac 02:42:df:27:05:00 \
   --b4 2001:db8::40,193.5.1.100,1024 --aftr 2001:db8:ffff::100 \
   --count 60000 --pcap lwaftr-traffic.pcap
$ tcpdump "ip" -r lwaftr-traffic.pcap -w from-inet.pcap
```

If wanting to add VLAN tags:

``` bash
$ tcprewrite --enet-vlan=add --enet-vlan-pri=0 --enet-vlan-cfi=0  \
   --enet-vlan-tag=444 --infile=from-inet.pcap --outfile=vlan/from-inet.pcap
```

**Comment by mwiget** [comment]
Sounds good. Consider the request for other packet sizes than IMIX as already done:

packetblaster lwaftr has a knob (--size) to specify a comma separated list fo packet sizes. 
@[https://github.com/Igalia/snabb/blob/lwaftr/src/program/packetblaster/lwaftr/README#L25-L32](url)

**Comment by wingo** [comment]
SGTM as well, nice idea.

**Comment by teknico** [comment]
Fixed by #508.
